src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import exception.EkudException;
+
 public class Duke {
     public static void main(String[] args) {
+        /*
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        */
+        System.out.println("I'm Ekud! What can I do for you?");
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        // Task[] tasks = new Task[100];
+        int taskCount = 0;
+        userInput = in.nextLine();
+
+        while(!userInput.equals("bye")){
+            String[] userInputWords = userInput.split(" ");
+
+            if(userInput.equals("list")){
```

Comment:
This if-else chain is **nested** and very **long**. This usually hints that there can be room for improvement for making your code [SLAP](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) more. 

To aid you in achieving SLAP, here are some guiding questions...
1. What parts of the code is repetitive? i.e. virtually the same for each if/else block you have
2. What part of the code does a particular function that can be isolated into a function?